sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/TypeScriptPlugin.java

```diff
@@ -21,33 +21,17 @@
 
 import org.sonar.api.Plugin;
 import org.sonar.api.SonarProduct;
-import org.sonar.api.config.PropertyDefinition;
-import org.sonar.api.resources.Qualifiers;
-import org.sonar.api.utils.Version;
 import org.sonar.plugin.typescript.executable.SonarTSCoreBundleFactory;
-import org.sonar.plugin.typescript.externalreport.TSLintRulesDefinition;
-import org.sonar.plugin.typescript.externalreport.TslintReportSensor;
-import org.sonar.plugin.typescript.lcov.LCOVCoverageSensor;
 
 public class TypeScriptPlugin implements Plugin {
   /* absolute location inside jar */
   private static final String SONARTS_BUNDLE_ZIP = "/sonarts-bundle.zip";
-  private static final String TESTS_AND_COVERAGE_SUBCATEGORY = "Tests and Coverage";
-  private static final String TYPESCRIPT_CATEGORY = "TypeScript";
-  private static final String GENERAL_SUBCATEGORY = "General";
-  private static final String EXTERNAL_ANALYZER_CATEGORY = "External Analyzers";
-  private static final String LINTER_SUBCATEGORY = "JavaScript/TypeScript";
 
   static final String FILE_SUFFIXES_KEY = "sonar.typescript.file.suffixes";
   public static final String FILE_SUFFIXES_DEFVALUE = ".ts,.tsx";
 
-  public static final String LCOV_REPORT_PATHS = "sonar.typescript.lcov.reportPaths";
-  public static final String LCOV_REPORT_PATHS_DEFAULT_VALUE = "";
-
   public static final String TSCONFIG_PATH = "sonar.typescript.tsconfigPath";
 
-  public static final String TSLINT_REPORT_PATHS = "sonar.typescript.tslint.reportPaths";
-  public static final String TSLINT_REPORT_PATHS_DEFAULT_VALUE = "";
 
   public static final String NODE_EXECUTABLE = "sonar.typescript.node";
```

Comment:
should we also rely on `sonar.nodejs.executable` in SonarTS? and rely on `sonar.typescript.node` in SonarJS?

Reply:
I think we should still support it, #1572